theme_tutorial/views/layout.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="layout" inherit_id="website.layout" name="Custom Header">
+        <xpath expr="//div[@id='wrapwrap']/header" position="after">
+            <div class="container">
```

Comment:
![image](https://github.com/AntonioMartinD/Building_a_website/assets/35231827/e3477e14-c461-4ec6-8e39-ab7616bbf1a7)

your element should be inside wrap